sp800-63a/sec4_ial.md

```diff
@@ -100,15 +98,12 @@ The following requirements apply to any CSP performing identity proofing at IAL2
 
 12. In the event a CSP ceases to conduct identity proofing and enrollment processes, the CSP SHALL be responsible for fully disposing of or destroying any sensitive data including PII, or its protection from unauthorized access for the duration of retention.
 
-13. Regardless of whether the CSP is an agency or private sector provider, the following requirements apply to the agency offering or using the proofing service:
-
-	a) The agency SHALL consult with their Senior Agency Official for Privacy (SAOP) to conduct an analysis to determine whether the collection of PII to conduct identity proofing triggers the requirements of the Privacy Act.
-
- 	b) The agency SHALL publish a System of Records Notice (SORN) to cover such collections, as applicable.
-
- 	c) The agency SHALL consult with their SAOP to conduct an analysis to determine whether the collection of PII to conduct identity proofing triggers the requirements of the E-Government Act of 2002.
+13. Regardless of whether the CSP is an agency or private sector provider, the following requirements apply to the agency offering or using the proofing service:  
 
- 	d) The agency SHALL publish a Privacy Impact Assessment (PIA) to cover such collections, as applicable.
+	a) The agency SHALL consult with their Senior Agency Official for Privacy (SAOP) to conduct an analysis to determine whether the collection of PII to conduct identity proofing triggers the requirements of the Privacy Act.  
+	b) The agency SHALL publish a System of Records Notice (SORN) to cover such collections, as applicable.  
```

Comment:
Here's a real nit: is "collections" correct here or should it be "collection" (the act of collecting)? Same in d). Collection seems more consistent with the text in other places, and the plural reads a little funny to me. But you can convince me otherwise.